tests/e2e/specs/in_game/oneOffs.spec.js

```diff
@@ -823,6 +822,112 @@ describe('Playing NINES', ()=>{
 				}
 			);
 		});
+
+		it('Plays a nine as a ONE-OFF, make sure that the bounced card is playable later', ()=>{
+			/*
+			1). P0 plays a 9, targeting an in-play Queen
+			2). P1 plays a 6's one-off, removing some cards
+			3). P0 plays a Jack, targeting an in-play 10
+			4). P1 couldn't replay their bounced Queen
+			 */
```

Comment:
With the bug fixed, rewrite step 4) to reflect the intended behavior, not the bug